skimage/filters/tests/test_gabor.py

```diff
@@ -77,6 +77,18 @@ def match_score(image, frequency):
     assert responses[1, 1] > responses[1, 0]
 
 
+def test_morlet_zero_sum():
```

Comment:
@siraferradans could you address this and the remaining minor comments, and then set the status of this PR to "needs review"?